Simulations/DALES/botany_dx100m.yaml

```diff
@@ -3,12 +3,6 @@ plugins:
     - module: intake_xarray
 
 sources:
-  nx768:
-    args:
-      path: "{{CATALOG_DIR}}/botany_dx100m_nx768.yaml"
-    description: 'Cloud botany LES ensemble at 100m grid spacing'
```

Comment:
"Cloud botany" is not very informative to outsiders, so perhaps this description could be expanded.